x-pack/test/functional/services/search_sessions.ts

```diff
@@ -72,7 +72,9 @@ export function SearchSessionsProvider({ getService }: FtrProviderContext) {
 
       if (searchSessionName) {
         await testSubjects.click('searchSessionNameEdit');
-        await testSubjects.setValue('searchSessionNameInput', searchSessionName);
+        await testSubjects.setValue('searchSessionNameInput', searchSessionName, {
+          clearWithKeyboard: true,
```

Comment:
In the original failure, the initial search session name wasn't cleared and a new name was appended instead.
`setValue` supports this different way of cleaning the input and according to flaky test runner - it is stable